src/runtime_src/core/tools/xbflash2/CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (C) 2022 Xilinx, Inc. All rights reserved.
+#
+
+find_package(Boost
+  REQUIRED COMPONENTS filesystem system program_options)
+
+include_directories(
+  ${CMAKE_CURRENT_SOURCE_DIR}
+  ${CMAKE_BINARY_DIR}/gen # includes version.h
+  )
+
+# Collect files outside of this directory
+file(GLOB XBFLASH_BASE_FILES
+  "../common/XBUtilitiesCore.cpp"
+  "../common/SubCmd.cpp"
+  "../common/OptionOptions.cpp"
+  "../common/XBHelpMenusCore.cpp"
+  "../../pcie/tools/xbflash.qspi/firmware_image.cpp"
```

Comment:
Just noticed this.   
Normally, we don't add periods '.' to directory names.  Instead we add an underscore char '_'.   Being that this was already added, please do consider refactoring the code post 2022.1 regarding this naming convention.

Reply:
noted this comment.